core/src/test/java/com/datastrato/gravitino/authorization/TestAccessControlManagerForPermissions.java

```diff
@@ -259,8 +270,11 @@ public void testDropRole() throws IOException {
             .build();
 
     entityStore.put(roleEntity, true);
-    Assertions.assertTrue(accessControlManager.grantRoleToUser(METALAKE, anotherRole, USER));
-    Assertions.assertTrue(accessControlManager.grantRoleToGroup(METALAKE, anotherRole, GROUP));
+
+    accessControlManager.grantRolesToUser(METALAKE, Lists.newArrayList(anotherRole), USER);
+
+    accessControlManager.grantRolesToGroup(METALAKE, Lists.newArrayList(anotherRole), GROUP);
```

Comment:
Should you check the return value of these two methods?

Reply:
Done.